typedoc.json

```diff
@@ -1,6 +1,6 @@
 {
 	"entryPoints": "src",
+	"exclude": "src/index.ts",
 	"includeVersion": true,
-	"readme": "CONTRIBUTING.md",
```

Comment:
Shouldn't we replace the readme field with README.md, since it does what CONTRIBUTING.md used to do? It's the only document that explains how to run our program currently.

Reply:
Yes, `README.md` is the default. It will appear in the front page of the docs site.